docs/installation/centos.md

```diff
@@ -56,7 +56,7 @@ package manager.
         $ cat >/etc/yum.repos.d/docker.repo <<-EOF
         [dockerrepo]
         name=Docker Repository
-        baseurl=https://yum.dockerproject.org/repo/main/centos/7
+        baseurl=https://yum.dockerproject.org/repo/main/centos/$releasever/
```

Comment:
This will evaluate `$releasever` at the time of the `cat`, pulling the value from the user's shell, which likely doesn't have the var, so it'd end up as `baseurl=https://yum.dockerproject.org/repo/main/centos//`.

Change the doc to `baseurl=https://yum.dockerproject.org/repo/main/centos/\$releasever/` (note the backslash) and you should be good.

Ditto for the other changes below.